R/dformat-oqr.R

```diff
@@ -519,8 +521,10 @@ les_dd_oqr = function(mappe_dd, reg_id, skjema_id, status = 1, dato = NULL, kode
     str_detect(x, "^-?[0-9]+(\\.[0-9]+)?$")
   }
   ## Ev. kjapp test på at funksjonen fungerer som han skal
+  # nolint start: commented_code_linter.
   # stopifnot(all(er_tal(c("-3", "0", "1", "997", "3.14", "-3.14", "0.7"))))
   # stopifnot(all(!er_tal(c("a", "2B", "F42.7", "-x", "1e-7", "3.", ".7")))) # Ev. godta "3." og .7"?
+  # nolint end
```

Comment:
Dette bør heller gjerast til ein formell testthat()-testa (`er_tal()` må då takast ut som ein eigen funksjon). Ev. finst det ein eigen talgjenkjenningsfunksjon i ein av pakkane me lastar, som me kan bruka?

Reply:
Einig, men det kan vel vera eit eige Trello-kort og PR?

Fann ikkje ein slik funksjon i `checkmate`-pakken. Fann funksjonen `varhandle::check.numeric()` som ser ut til å gjera det me er ute etter? Men veit ikkje om me vil ha inn endå ein pakke berre for dette.

Reply:
Nei, ikkje ny pakke inn. Du lagar Trello-kort?